components/ww3/cime_config/config_compsets.xml

```diff
@@ -24,6 +24,11 @@
     <lname>2000_DATM%CFSR_SLND_SICE_SOCN_SROF_SGLC_WW3%sp36x36</lname>
   </compset>
 
+  <compset>
+    <alias>DTESTM-WW3</alias>
```

Comment:
`DTESTM` is used in D-cases with CORE NYF. I'd suggest renaming the alias `DMPAS-JRA-WW3` to have parity with the G-case equivalent here: https://github.com/E3SM-Project/E3SM/pull/5274.

Reply:
great suggestion - thanks!